src/analysis/analyzer_gen.ts

```diff
@@ -8,7 +8,7 @@ import * as as from "./analysis_server_types";
 import * as a from "./analyzer";
 
 export abstract class AnalyzerGen {
-	protected abstract notify<T>(subscriptions: ((notification: T) => void)[], params: T);
+	protected abstract notify<T>(subscriptions: ((notification: T) => void)[], params: T): void;
```

Comment:
:memo: This appears to be in generated code, but I don't know how to update the original source.